docs/100-upgrade/migration-guides/3.7-3.8.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 3.7 -> 3.8
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.7
+  to 3.8
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/*@3.8.0"
+```
+
+## Automated Migration
+
+We provide a codemod to automatically update your codebase to the latest version
+of Front-Commerce. This tool will update your code when possible and flag the
+places where you need to manually update your code (with `// TODO Codemod`
+comments).
+
+```shell
+pnpm run front-commerce migrate --transform 3.8.0
+```
+
+## Additional SEO meta data
```

Comment:
**thought:** this whole `Additional SEO meta` section doesn't contain any actionable information for the integrator. Is it really meant to be in the migration guide?